### PR TITLE
Add logstash write timer metric

### PR DESCRIPTION
**Pull request by tomez**

### CODE REVIEW

File: servicelog/appender/logstash.go [hunk 1]
```diff
@@ -78,7 +79,8 @@ func (l *logstash) sendEntry(entry servicelog.Entry) error {
 		return fmt.Errorf("unable to marshal log entry: %s", err)
 	}
 	log.WithField("entry", string(bytes)).Debug("Sending log entry to Logstash")
-	if _, err = l.writer.Write(bytes); err != nil {
+	l.writeTimer.Time(func() { _, err = l.writer.Write(bytes) })
+	if err != nil {
```
Comment (janisz) [inline servicelog/appender/logstash.go]: `err` is always `nil` here, it was checked before. Error handling should be done in the function above.

File: servicelog/scraper/json.go [hunk 2]
```diff
@@ -11,6 +10,8 @@ import (
 	"github.com/rcrowley/go-metrics"
 	log "github.com/sirupsen/logrus"
 
+	"fmt"
```
Comment (janisz) [inline servicelog/scraper/json.go]: Sort and group imports

File: servicelog/appender/logstash.go (hunk 1 above)
Comment (tomez) [reply servicelog/appender/logstash.go]: I don't follow, err was checked before - true, but there is another call (wrapped) after that.

Comment (janisz) [reply servicelog/appender/logstash.go]: OK, my bad

**Pull request merged**